doc/common_info.md

```diff
@@ -521,3 +521,23 @@ exec "$bazel_real" "$@"
 ```
 
 In your main `.bazelrc` add `import xcode.bazelrc` at the very bottom.
+
+## Optimizing remote build and cache performance
+
+When using Bazel's remote cache and/or build execution, there are a few flags you can pass to optimize performance.
+
+One of these is [`modify_execution_info`](https://bazel.build/reference/command-line-reference#flag--modify_execution_info).
+This flag allows adding or removing [mnemonics](https://bazel.build/reference/glossary#mnemonic) to configure what is cached or built remotely.
+
+We recommend adding the following to your `.bazelrc`:
+
+```shell
+common --modify_execution_info=^(BitcodeSymbolsCopy|BundleApp|BundleTreeApp|DsymDwarf|DsymLipo|GenerateAppleSymbolsFile|ObjcBinarySymbolStrip|CppLink|ObjcLink|ProcessAndSign|SignBinary|SwiftArchive|SwiftStdlibCopy)$=+no-remote,^(BundleResources|ImportedDynamicFrameworkProcessor)$=+no-remote-exec
+```
+
+The following table provides a rationale for each mnemonic. In general though, the mnemonics that are excluded in `modify_execution_info` are excluded because they are large outputs which change frequently and as such are faster when run locally, or they are not generally configured for remote execution (such as signing).
+
+| Mnemonic | Rationale |
+| --- | --- |
+|ProcessAndSign | A signing action, RBE is not generally configured for signing. |
+|SignBinary | A signing action, RBE is not generally configured for signing. |
```

Comment:
Light opinion: for the sake of brevity, I like Brentley and Keith's approach of reversing this and saying:

<Reason>: <List of mnemonic's that map to this reason>